pyattck/pyattck.py

```diff
@@ -16,8 +16,15 @@ class Attck(object):
         [Attck]: Returns a Attck object that contains all data from the Mitre ATT&CK Framework
     """
 
-    def __init__(self):
-        self.attck = __MITRE_ATTCK_JSON_URL__
+    def __init__(self, local_file=False):
```

Comment:
I would change this to `local_file_path` and set it to `None` as the default

Reply:
Then if `local_file_path` has a value then we should create it.  Also, set a new `self.local_file_path` variable here with the value which can be used in other methods within this class.